hash.go

```diff
@@ -0,0 +1,46 @@
+package wolfSSL
+
+// #cgo CFLAGS: -g -Wall -I/usr/include -I/usr/include/wolfssl
+// #cgo LDFLAGS: -L/usr/local/lib -lwolfssl -lm
+// #include <wolfssl/options.h>
+// #include <wolfssl/wolfcrypt/hash.h>
+// #undef NO_MD5
+//
+// #ifdef NO_MD5
+// #define WC_MD5_DIGEST_SIZE 1
+// int wc_Md5Hash(const byte* data, word32 len, byte* hash) {
+//      return -999;
```

Comment:
Like you suggested, let's switch `-999` to `NOT_COMPILED_IN` (-174) to be consistent with wolfCrypt/wolfSSL.